packages/keeper/scripts/jobs.js

```diff
@@ -0,0 +1,27 @@
+const { Worker } = require('bullmq');
+const config = require('../src/config');
+
+async function mintsProcessor(job) {
+  // TODO: Get job data and if a new batch was triggered, upload the necessary
+  //       resources to the IPFS server.
```

Comment:
On a following PR, here I will add the necessary logic for uploading the assets.